stl/inc/chrono

```diff
@@ -2209,6 +2221,2946 @@ namespace chrono {
+        }
+    }
+
+    // CLASS utc_clock
+    class utc_clock {
+    public:
+        using rep                       = system_clock::rep;
+        using period                    = system_clock::period;
+        using duration                  = duration<rep, period>;
+        using time_point                = time_point<utc_clock>;
+        static constexpr bool is_steady = system_clock::is_steady;
+
+        _NODISCARD static time_point now() {
+            return from_sys(system_clock::now());
+        }
+
+        template <class _Duration>
+        _NODISCARD static sys_time<common_type_t<_Duration, seconds>> to_sys(const utc_time<_Duration>& _Utc_time) {
+            using _CommonType = common_type_t<_Duration, seconds>;
+            const auto _Lsi{_CHRONO get_leap_second_info(_Utc_time)};
+            _CommonType _Ticks;
+            if (_Lsi.is_leap_second) {
+                const auto _Leap_sec_minus_one = _CHRONO floor<seconds>(_Utc_time.time_since_epoch()) - _Lsi.elapsed;
+                if constexpr (is_integral_v<typename _Duration::rep>) {
+                    constexpr auto _Delta{seconds{1} - _CommonType{1}};
+                    _Ticks = _Leap_sec_minus_one + _Delta;
+                } else {
+                    const auto _Leap_sec_begin = _CHRONO ceil<_CommonType>(_Leap_sec_minus_one + seconds{1});
+                    _Ticks = _CommonType{_STD nextafter(_Leap_sec_begin.count(), typename _CommonType::rep{0})};
+                }
+            } else {
+                _Ticks = _Utc_time.time_since_epoch() - _Lsi.elapsed;
+            }
+            return sys_time<_CommonType>{_Ticks};
+        }
+
+        template <class _Duration>
+        _NODISCARD static utc_time<common_type_t<_Duration, seconds>> from_sys(const sys_time<_Duration>& _Sys_time) {
+            const auto& _Tzdb      = _CHRONO get_tzdb();
+            const auto& _Ls_vector = _Tzdb.leap_seconds;
+            auto _It = _STD upper_bound(_Ls_vector.begin(), _Ls_vector.end(), _CHRONO floor<seconds>(_Sys_time));
+            const auto _Offset = _It == _Ls_vector.begin() ? seconds{0} : (--_It)->_Elapsed();
+            return utc_time<common_type_t<_Duration, seconds>>{_Sys_time.time_since_epoch() + _Offset};
+        }
+    };
+
+    // [time.clock.tai]
+
+    class tai_clock;
+
+    template <class _Duration>
+    using tai_time    = time_point<tai_clock, _Duration>;
+    using tai_seconds = tai_time<seconds>;
+
+    // CLASS tai_clock
+    class tai_clock {
+    public:
+        using rep                       = system_clock::rep;
+        using period                    = system_clock::period;
+        using duration                  = duration<rep, period>;
+        using time_point                = time_point<tai_clock>;
+        static constexpr bool is_steady = system_clock::is_steady;
+
+        static constexpr seconds _Tai_epoch_adjust{378691210};
+
+        _NODISCARD static time_point now() noexcept(noexcept(utc_clock::now())) {

```

Comment:
This would need to be commented as strengthened. However, since we haven't strengthened `utc_clock::now()` above, I think this should simply be dropped. Ditto for `gps_clock`.